internal/drain-config/drainConfig.go

```diff
@@ -0,0 +1,179 @@
+package drain_config
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+	"slices"
+	"strings"
+	"time"
+
+	"github.com/samber/lo"
+
+	"github.com/rudderlabs/rudder-go-kit/config"
+	"github.com/rudderlabs/rudder-go-kit/logger"
+	migrator "github.com/rudderlabs/rudder-server/services/sql-migrator"
+	"github.com/rudderlabs/rudder-server/utils/misc"
+)
+
+const (
+	defaultPollFrequency      = 5
+	defaultPollFrequencyUnits = time.Second
+
+	defaultCleanupFrequency      = 24
```

Comment:
Let's run cleanup more frequently
```suggestion
	defaultCleanupFrequency      = 1
```